### PR TITLE
fixes #2363 Selected compute resource provider does not remain selected on failure

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
@ohadlevy rebased and ready for merge

**Comment by ohadlevy** [comment]
@isratrade interesting, do you have an idea why you had to do that?

**Comment by isratrade** [comment]
@ohadlevy, I forget which case this was required.  I just tried without the change and it was OK, but I know there was a case where it was needed.

**Comment by isratrade** [comment]
@ohadlevy not sure by this pr showed failing tests on jenkins.  rebased

File: app/views/compute_resources/_form.html.erb [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <% end %>
         <%= text_f f, :name %>
         <%= f.hidden_field(:provider, :id => '') if f.object.uuid.present? %>
-        <%= selectable_f f, :provider, ComputeResource::PROVIDERS, { :include_blank => "Choose a provider"},
+        <%= selectable_f f, :provider, ComputeResource::PROVIDERS, { :include_blank => "Choose a provider", :selected => f.object.provider},
```
Comment (ohadlevy) [inline app/views/compute_resources/_form.html.erb:21]: I'm not following why this is required, as the selectable helper should use the default value?

**Comment by isratrade** [comment]
@ohadlevy, I'm closing this pr.  I can't reproduce why it was need.

**Pull request closed without merging**